### DISCUSSION TITLE
fullBuild and impactBuild continue even in the case of an error?

### DISCUSSION THREAD

**Discussion by MarcBauer-LM** [IBM/dbb-zappbuild#444] [Q&A] [answered]
As far as I understand it, the full build continues even in the event of an error. This results in a failed build that has to be repeated in any case. So why is the process not canceled after the first failure? Is there a reason for this behavior or am I missing something?

**Comment by dennis-behm** [comment, marked as answer]
Your observations are correct. We have a pending enhancement request for that #257 , which would address your concern.
